clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/converters/CloneCloudFoundryServerGroupAtomicOperationConverter.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Why is a separate operation necessary if it extends from and doesn't modify the behavior of `DeployCloudFoundryServerGroupAtomicOperationConverter`?

Reply:
Core Deck determines that you execute a clone action and puts that operation string on it. As a result you have to have a handler for it on the receiving end in clouddriver.